docs/source/autograd.rst

```diff
@@ -7,9 +7,12 @@ Automatic differentiation package - torch.autograd
 .. automodule:: torch.autograd
 .. currentmodule:: torch.autograd
 
-.. autofunction:: backward
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
 
-.. autofunction:: grad
+    backward
+    grad
```

Comment:
Is there a way to fix the following? It should say "Computes the sum of gradients of given tensors w.r.t. graph leaves."

![image](https://user-images.githubusercontent.com/5652049/114422919-b20b3780-9b84-11eb-963a-e2f9980ae4d6.png)

Reply:
I tried spelling out w.r.t., maybe the period breaks formatting?

Reply:
fixed https://12338049-65600975-gh.circle-artifacts.com/0/docs/autograd.html